groups/bdl/bdldfp/bdldfp_decimalutil.t.cpp

```diff
@@ -427,13 +434,267 @@ struct NulBuf : bsl::streambuf {
 //                              MAIN PROGRAM
 //-----------------------------------------------------------------------------
 
+
+static void testCase13()
```

Comment:
Why is this test case factored out of the `main` switch statement?

We did that in STL container components (since the test itself was instantiated on multiple types), and we'll sometimes do that when a test driver main becomes so large a compiler has trouble compiling/optimizing `main` in a reasonable amount of time.  But, all things being equal, test-cases should be inline in the body of `main`